src/lib/index.ts

```diff
@@ -45,8 +45,12 @@ class AssetEntry {
         }
     }
     private applyScript(tag: HTMLScriptElement) {
+        const start = /<script>/gi;
+        const end = /<\/script>gi;
         tag.removeAttribute('src');
-        tag.innerHTML = this.content.toString();
+        var js = this.content.toString();
```

Comment:
Do we use `js` variable?

Reply:
sorry - thats a mistake, i had to retype the fix from my work computer to my personal one, updating the fix now